apps/tx-builder/src/components/ModalBody.tsx

```diff
@@ -1,10 +1,16 @@
-import { Button, Text } from '@gnosis.pm/safe-react-components';
 import React from 'react';
+import { Button, Text } from '@gnosis.pm/safe-react-components';
 import Box from '@material-ui/core/Box';
+import styled from 'styled-components';
 import { ProposedTransaction } from '../typings/models';
 
 type Props = { txs: Array<ProposedTransaction>; deleteTx: (index: number) => void };
 
+const WrappedText = styled(Text)`
+  word-break: break-all;
+  margin: 5px 0;
```

Comment:
should we use spacings from the theme instead of hardcoded values?

Reply:
Good idea, i can do it. BTW this margin/paddings are mostly hardcoded in all the apps  so perhaps we should add an improvement for cleaning this type of things

Reply:
So, I think is better to leave it like this as in all the safe apps there are fixed values instead of using the ones in the theme. We are using as well a direct commit in this app instead the version and we don't have access to the `margin` prop for using predefined spacing

https://github.com/gnosis/safe-react-apps/blob/development/apps/tx-builder/package.json#L7

So, need to update but ... the latest version have a bug with the checkboxes as well being addressed currently but not yet released.

My suggestion would be to resolve this PR with the fixed value and create a ticket for updating the `safe-react-components` package to the latest version and use the theme properly instead fixed values. What do you think @dasanra ?

Reply:
For me it's correct right now to resolve this issue like this and to improve the `safe-react-components` to provide theming.

Bigger break in `safe-react-components` can be caused on web. External apps are not using this library in a very heavy way, although we should make clear this changes in the changelog and with semantic versioning